### PR TITLE
Addition of database creation and seeding sql

### PR DESCRIPTION
**Pull request by williamkimball**
# Description
This PR includes the addition of a file called ```BangazonDatabaseCreateSeed.sql``` that contains the code to create a database and seed it with the information needed for this project.
 Fixes # 23
 ## Type of change
- [x] This change requires a documentation update
 # Testing Instructions

Clone the Obedient Lobsters WorkforceManagement repo onto your machine. ```cd``` into that directory and open the project in Visual Studio Code.
Make sure the database is built on your local machine using the contents of ```BangazonDatabaseSeedBuild``` in the project root directory (see database setup above for instructions).
Link Visual Studio Code to that database by going to ```View``` and selecting the ```SQL Server Object Explorer```. Open that up and press the ```add SQL server``` button (looks like a column with a green plus). Then select ```local``` and pick the option that matches your local server.

If all went correctly, your database should be connected, and you can then run the project.
To run the project, press the green "play" triangle that is above the code editor, roughly in the middle.
Check to make sure that the project still builds and runs.
 # Checklist:
 - [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: WorkforceManagement/WorkforceManagement/BangazonDatabaseCreateSeed.sql [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+  EmployeeTrainingId  INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  TrainingProgramId   INTEGER NOT NULL,
+  EmployeeId          INTEGER NOT NULL,
+  CONSTRAINT FK_TrainingProgram FOREIGN KEY(TrainingProgramId)REFERENCES TrainingProgram(TrainingProgramId),
+  CONSTRAINT FK_EmployeeTraining FOREIGN KEY(EmployeeId)REFERENCES Employee(EmployeeId)
+);
+CREATE TABLE EmployeeComputer (
+  EmployeeComputerId  INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  DateAssigned        DATE NOT NULL,
+  DateReturned        DATE NOT NULL,
```
Comment (Shu52) [inline WorkforceManagement/WorkforceManagement/BangazonDatabaseCreateSeed.sql]: Please change DateReturned from DATE NOT NULL to DATE.

Comment (williamkimball) [reply WorkforceManagement/WorkforceManagement/BangazonDatabaseCreateSeed.sql]: It is finished

**Comment by Rian501** [comment]
This pull request appears to contain more than it advertises. The build does run without issues, but can you please either include the additional files in your PR description/title or remove them from this particular PR?

**Comment by Rian501** [comment]
The SQL looks good. You may want to seed with some training programs that have not yet happened, and some computer-employee relationships that have not yet ended, but otherwise that part is a-ok.

**Pull request closed without merging**